src/test/regress/expected/multi_insert_select_window.out

```diff
@@ -709,8 +709,8 @@ SELECT * FROM
 ) as foo
 LIMIT
   10;
-ERROR:  could not run distributed query with window functions
-HINT:  Consider using an equality filter on the distributed table's partition column.
+ERROR:  could not run distributed query becuase the window function that is used cannot be pushed down
+HINT:  Window functions are supported in two ways. Either add an quality filter on the distributed tables' partition columns or use the window functions inside a subquery with PARTITION BY clause containing the distribution column
```

Comment:
quality -> equality ?

unless you mean "add a really good filter" ;)

partition columns -> partition column

with PARTITION BY clause -> with a PARTITION BY clause